web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -99,33 +92,87 @@ export default function ZoneDetails(): JSX.Element {
           zoneDataStatus !== ZoneDataStatus.AGGREGATE_DISABLED && (
             <DisplayByEmissionToggle />
           )}
-        <ZoneDetailsContent
-          isLoading={isLoading}
-          isError={isError}
-          zoneDataStatus={zoneDataStatus}
-        >
-          <BarBreakdownChart hasEstimationPill={hasEstimationPill} />
-          <CommercialApiButton backgroundClasses="mt-3 mb-1" type="link" />
-          {zoneDataStatus === ZoneDataStatus.AVAILABLE && (
-            <AreaGraphContainer
-              datetimes={datetimes}
-              timeAverage={timeAverage}
-              displayByEmissions={displayByEmissions}
-            />
-          )}
-          <MethodologyCard />
-          <Attribution zoneId={zoneId} />
-          {isMobile ? (
-            <CommercialApiButton backgroundClasses="mt-3" />
-          ) : (
-            <div className="p-2" />
-          )}
-        </ZoneDetailsContent>
+        <ZoneDetailRoutes />
       </div>
     </>
   );
 }
 
+export function ZoneDetailRoutes() {
+  return (
+    <Routes>
+      <Route index element={<ZoneDetailsContent />} />
+      <Route path={LeftPanelToggleOptions.ELECTRICITY} element={<ZoneDetailsContent />} />
+      <Route path={LeftPanelToggleOptions.EMISSIONS} element={<ZoneDetailsContent />} />
+    </Routes>
+  );
+}
```

Comment:
Not sure how I feel about this. We got to be real careful when we structure these URLs as they will basically live forever.

If we add electricity/emissions directly to the url at the end of the path I'm unsure how that plays into the deeper work we need to do around the URL structure for historical linking and consumption/production modes.

We would also need to add it to the sitemap and ensure it works with the canonical URL properly.

Reply:
💯. This draft PR was just to test some things out for the chart linking work. If we choose to pursue this avenue, then we would probably want to write up a proper brief.